slave/components/quality_controller/error_correction.go

```diff
@@ -96,5 +96,5 @@ func (e *errorCorrection) Process() (string, error) {
 	if _, err := io.Copy(os.Stdout, out); err != nil {
 		return "", fmt.Errorf("failed to capture stdout from Docker assembly container, err: %v", err)
 	}
-	return correctedFile, nil
+	return path.Join(e.config.GenoMagic.OutputPath, "canu-corr", correctedOutuptFile), nil
```

Comment:
Is `canu-corr` the output directory containing the corrected file? Any chance we can pass our own filename? 🤷‍♂️